docs/atlascli/command/atlas-backups-restores-list.txt

```diff
@@ -85,3 +85,11 @@ Inherited Options
      - false
      - Profile to use from your configuration file.
 
+Examples
+--------
+
+.. code-block::
+
+ The following example etrieve the Continuous Backup restore jobs for the cluster Cluster0:
```

Comment:
`retrieves` not etrieve.

Also, do we somehow mark names, e.g. **Cluster0** or *Cluster0*?

Reply:
I'll let @mongodb/docs-cloud-team decide on formatting, I think I copy pasted the text from an old mongocli docs page